src/pages/Login/Login.js

```diff
@@ -1,7 +1,90 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 import './Login.scss';
+import Sns from './Sns';
+
 const Login = () => {
-  return <div />;
+  const [person, setPerson] = useState({
+    id: '',
+    pw: '',
+  });
+
+  const handleInput = e => {
+    const { name, value } = e.target;
+    setPerson({ ...person, [name]: value });
+  };
+  // 아이디, 비밀번호 정규식
+  const passwordRegExp =
+    /^(?=.*[a-z])(?=.*\d)(?=.*[$@$!%*#?&])[a-z\d$@$!%*#?&]{8,16}$/;
+  const idRegExp = /^[a-z]+[a-z0-9]{4,12}$/;
+
+  const validation = idRegExp.test(person.id) && passwordRegExp.test(person.pw);
+
+  const navigate = useNavigate();
+
+  const signIn = e => {
+    e.preventDefault();
+    fetch('http://10.58.5.132:8000/users/login', {
+      method: 'POST',
+      body: JSON.stringify({
+        username: person.id,
+        password: person.pw,
+      }),
+    })
+      .then(response => response.json())
+      .then(result => {
+        if (result.message === 'SUCCESS') {
+          navigate('/');
+          alert('로그인 성공!');
+        }
+      });
+  };
+
+  return (
+    <div>
+      <div className="login-header">
+        <div className="login-nav">
+          <h1>로그인</h1>
+          <button type="submit">
+            <i class="fa-solid fa-xmark" />
+          </button>
+        </div>
+      </div>
+      <section className="body">
+        <div className="login-container">
+          <div className="header">
+            <img src="../images/login/LocTea.png" alt="" />
```

Comment:
img alt값 올바르게 작성해주세요! 그리고 public 폴더 하위의 이미지는 절대경로로 접근하실 수 있습니다.
```suggestion
            <img src="/images/login/LocTea.png" alt="" />
```

Reply:
img alt값 없는 다른 부분도 추가해주세요!